covalent/executor/executor_plugins/timing_plugin.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 Agnostiq Inc.
```

Comment:
We shouldn't be adding this file to covalent's codebase but rather in the How Tos. The entire purpose of this executor's existence is that it is a user defined executor and thus will not be a part of covalent when it is shipped and only added during execution time.

Reply:
Hmm ok I see